Bindings/Java/Matlab/configureOpenSim.m.in

```diff
@@ -39,7 +39,7 @@
             error('User cancelled function')
         end
         % Check if the correct OpenSim directory is selected
-        if ~exist(fullfile(openSimFolder, '@CMAKE_INSTALL_SYSCONFDIR@', 'buildinfo.txt'))
+        if ~exist(fullfile(openSimFolder, '@CMAKE_INSTALL_SYSCONFDIR@', 'OpenSim_buildinfo.txt'))
```

Comment:
I am trying to handle the use case where someone installs OpenSim into a system wide directory like `/usr` or `/usr/local`. There would not be sufficient context in `/usr/local/etc/` (SYSCONFDIR is `etc`).

I wouldn't expect multiple buildinfo.txt files, but it would not be obvious what the file is.